styles.css

```diff
@@ -0,0 +1,36 @@
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
I will suggest you also include box-sizing property here and set it to border-box.

```
box-sizing: border-box

```
This ensures that the specified width and height of the elements correspond with what is stated and that no other extra padding is included in the specified sizes. 
To learn more about these you can take a look at this link https://www.w3schools.com/css/css3_box-sizing.asp